elf_reader.go

```diff
@@ -27,6 +27,7 @@ type elfCode struct {
 	version  uint32
 	btf      *btf.Spec
 	extInfo  *btf.ExtInfos
+	maps     *map[string]*MapSpec
```

Comment:
```suggestion
	maps     map[string]*MapSpec
```
Go maps are already pointer types and nil by default, no need to take out pointers to them.